asdf_astropy/resources/schemas/fits/fitswcs-1.0.0.yaml

```
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/fits/fitswcs-1.0.0"
+
+title:
+  Represents the fits object
```

```suggestion
title: Represents the fits object
```

Would you add a more descriptive title?